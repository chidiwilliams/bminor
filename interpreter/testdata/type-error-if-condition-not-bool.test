f: integer = 0;
if (f) {
    print f;
}
