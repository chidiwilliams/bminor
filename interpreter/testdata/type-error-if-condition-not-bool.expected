err: Error: 'f' must be a boolean
